rpmbuild/copr_rpmbuild/config.py

```diff
@@ -0,0 +1,37 @@
+"""
+Configuration class for copr-rpmbuild
+"""
+
+
+import os
+import yaml
+
+
+CONFIG_PATH = "/etc/copr-rpmbuild/copr-rpmbuild"
+ALLOWED_FILE_TYPES = ["yaml", "yml"]
+
+
+class Config:
+    """
+    Configuration class for copr-rpmbuild
+    """
+    def __init__(self):
+        self.job_tag_to_mock_snippets = {}
+
+    def load_config(self):
+        """
+        Load configuration from the config file
+        """
+        config_path = None
+        for file_type in ALLOWED_FILE_TYPES:
+            if os.path.exists(f"{CONFIG_PATH}.{file_type}"):
```

Comment:
Use of `os.path.exists` ;) it's an antipattern, you just do two syscalls without reason.  Just open the file, done.